testsuite/tests/libratest/smoke_test.rs

```diff
@@ -1174,3 +1185,65 @@ fn test_malformed_script() {
         .mint_coins(&["mintb", "0", "10"], true)
         .unwrap();
 }
+
+#[test]
+fn test_key_manager_consensus_rotation() {
+    // Create and launch a local validator swarm of 2 nodes.
+    let mut swarm = TestEnvironment::new(2);
+    swarm.launch_swarm(RoleType::Validator);
+
+    // Create a node config for the key manager by modifying the first node config in the swarm.
+    let config_path = swarm.validator_swarm.config.config_files.get(0).unwrap();
+    let mut config = NodeConfig::load(&config_path).unwrap();
+    let json_rpc_endpoint = format!("http://127.0.0.1:{}", config.rpc.address.port());
+    let backend = config.consensus.safety_rules.backend.clone();
+    config.secure.key_manager.json_rpc_endpoint = json_rpc_endpoint.clone();
+    config.secure.key_manager.secure_backend = backend.clone();
+    config.secure.key_manager.rotation_period_secs = 15;
+    config.secure.key_manager.sleep_period_secs = 15;
+    config.save(config_path).unwrap();
+
+    // Bootstrap secure storage by initializing the keys required by the key manager.
+    // TODO(joshlind): FIX ME -- these keys should already be set when the validator is initialized.
```

Comment:
David, do you think it's worth trying to modify the code that initializes these keys in this commit? It's unclear to me how we expect secure storage to be initialized (e.g., through LSR, tooling etc.)

Reply:
You have to initialize your keys prior to genesis, part 2. So this isn't horrible, but we should probably align this with some sort of functionality in config/management. Maybe we move the initialize into a public function? Though we probably have to live with this until config/management has a smoke test.

Reply:
Thanks, I've added a TODO. I can come back to this once config/management has support.